src/main/java/org/broadinstitute/consent/http/service/SummaryService.java

```diff
@@ -47,43 +48,31 @@
 /**
  * Implementation class for VoteAPI on top of ElectionDAO database support.
  */
-public class DatabaseSummaryAPI extends AbstractSummaryAPI {
-
-    private VoteDAO voteDAO;
-    private ElectionDAO electionDAO;
-    private UserDAO userDAO;
-    private ConsentDAO consentDAO;
-    private DataSetDAO datasetDAO;
-    private MatchDAO matchDAO;
-    private DataAccessRequestService dataAccessRequestService;
+public class SummaryService {
+
+    private final VoteDAO voteDAO;
+    private final ElectionDAO electionDAO;
+    private final UserDAO userDAO;
+    private final ConsentDAO consentDAO;
+    private final DataSetDAO datasetDAO;
+    private final MatchDAO matchDAO;
+    private final DataAccessRequestService dataAccessRequestService;
     private static final String SEPARATOR = "\t";
     private static final String TEXT_DELIMITER = "\"";
     private static final String END_OF_LINE = System.lineSeparator();
     private static final String MANUAL_REVIEW = "Manual Review";
     private final Logger logger = LoggerFactory.getLogger(this.getClass());
 
-
-    /**
-     * Initialize the singleton API instance using the provided DAO. This method
-     * should only be called once during application initialization (from the
-     * run() method). If called a second time it will throw an
-     * IllegalStateException. Note that this method is not synchronized, as it
-     * is not intended to be called more than once.
-     *
-     * @param dao The Data Access Object instance that the API should use to
-     *            read/write data.
-     */
-    public static void initInstance(DataAccessRequestService dataAccessRequestService, VoteDAO dao, ElectionDAO electionDAO, UserDAO userDAO, ConsentDAO consentDAO, DataSetDAO datasetDAO, MatchDAO matchDAO) {
-        SummaryAPIHolder.setInstance(new DatabaseSummaryAPI(dataAccessRequestService, dao, electionDAO, userDAO, consentDAO, datasetDAO, matchDAO));
-    }
-
     /**
      * The constructor is private to force use of the factory methods and
```

Comment:
Is this comment outdated since the constructor is not public rather than protected?

Reply:
Yes, good catch!